src/components/scenes/TransactionListScene.tsx

```diff
@@ -71,7 +71,7 @@ function TransactionListComponent(props: Props) {
 
   // Watchers:
   const enabledTokenIds = useWatch(wallet, 'enabledTokenIds')
-  const transactionList = useTransactionList(wallet, tokenId, isSearching ? searchText : undefined)
+  const transactionList = useTransactionList(wallet, tokenId, isSearching ? searchText : '')
```

Comment:
Does the glitchy behavior happen when blur-ing the input?

Also, would prefer to see this default `''` encoded into `useTransactionList` instead so future users don't need to know about this quirk.

Reply:
I moved the fix into the hook